lighthouse-extension/app/src/lightrider-entry.js

```diff
@@ -46,6 +47,14 @@ async function runLighthouseInLR(connection, url, flags, {lrDevice, categoryIDs,
     if (logAssets) {
       await assetSaver.logAssets(results.artifacts, results.lhr.audits);
     }
+
+    // pre process the LHR for proto
+    if (flags.output === 'json') {
```

Comment:
if (flags.output === 'json' && typeof results.report === 'string')

Reply:
actually instead...

call `preprocessor.processForProto(results.lhr);`  and avoid the parse inside of that method?